mlflow/tracing/export/mlflow_v3.py

```diff
@@ -131,3 +155,158 @@ def _should_log_async(self):
+            )
+            return False
+        
+        # Ingestion URL is always required regardless of auth method
+        if not MLFLOW_TRACING_DELTA_ARCHIVAL_INGESTION_URL.get():
+            _logger.warning(
+                f"Delta archival is enabled but {MLFLOW_TRACING_DELTA_ARCHIVAL_INGESTION_URL.name} is not set. "
+                "Disabling delta archival."
+            )
+            return False
+        
+        # Priority 1: Direct token override - if token is explicitly set, use it
+        if MLFLOW_TRACING_DELTA_ARCHIVAL_TOKEN.get():
+            _logger.debug("Using direct token override for delta archival")
+            return True
+        
+        # Priority 2: Try standard Databricks authentication
+        try:
+            get_databricks_host_creds("databricks")
+            return True
+        except Exception as e:
+            _logger.debug(f"Standard Databricks authentication failed: {e}")
+        
+        # Priority 3: Final fallback to legacy custom environment variables
+        # Only workspace URL is needed since token and ingestion URL are already checked above
+        if not MLFLOW_TRACING_DELTA_ARCHIVAL_WORKSPACE_URL.get():
+            _logger.warning(
+                "Delta archival is enabled but no valid authentication method found. "
+                f"Missing: {MLFLOW_TRACING_DELTA_ARCHIVAL_WORKSPACE_URL.name}. "
+                "Disabling delta archival."
+            )
+            return False
+        
+        return True
+
+    def _resolve_token(self) -> Optional[str]:
+        """
+        Resolve authentication token using priority order:
+        1. Direct token override
+        2. Standard Databricks authentication
+        3. Return None if no token available
+        """
+        # Priority 1: Direct token override
+        if MLFLOW_TRACING_DELTA_ARCHIVAL_TOKEN.get():
+            _logger.debug("Using direct token override")
+            return MLFLOW_TRACING_DELTA_ARCHIVAL_TOKEN.get()
+        
+        # Priority 2: Standard Databricks authentication
+        try:
+            host_creds = get_databricks_host_creds("databricks")
+            if host_creds and host_creds.token:
+                _logger.debug("Using token from standard Databricks authentication")
+                return host_creds.token
+        except Exception as e:
+            _logger.debug(f"Standard Databricks authentication failed: {e}")
+        
+        # No valid token found
+        _logger.debug("No valid authentication token found")
+        return None
+
+    def _resolve_workspace_url(self) -> Optional[str]:
+        """
+        Resolve workspace URL using priority order:
+        1. From custom environment variable (takes priority)
+        2. From standard Databricks authentication (host)
+        3. Return None if no URL available
+        """
+        # Priority 1: Custom environment variable takes priority
+        workspace_url = MLFLOW_TRACING_DELTA_ARCHIVAL_WORKSPACE_URL.get()
```

Comment:
No need for custom workspaces / tokens, IMO. We should reuse the MLflow Tracking workspace URL and token.